index.html

```diff
@@ -1,202 +1,503 @@
+	<link rel="stylesheet" href="./css/normalize.css" />
+	<link rel="stylesheet" href="css/global.css" />
+	<!-- linked to created home.css -->
+	<link rel="stylesheet" href="./css/home.css" />
+</head>
+
+<body>
+	<!-- header section -->
+
+	<!-- =======================
+		Hero Section
+		========================== -->
+
+	<!-- "hero" is the class for the entire section -->
+	<section class="hero">
+		<!-- image of girl with dandelion -->
+		<!-- this image is only displayed on mobile and does not use a source set like the other sections -->
+		<img
+			src="./assets/images/pages/home/mobile/hero-home-mobile-2.jpg"
+			alt="a little girl holding a flower" />
+
+		<!-- "hero-home" holds the heading, paragraph and buttons for the hero image section -->
+		<!-- global class of "wrapper" is applied -->
+		<!-- "align-text" is a home-page class to center the text -->
+		<div class="hero-home wrapper align-text">
+			<h3>Preserving Tomorrow by Building Today</h3>
+			<p>
+				If These Lands Could Talk (ITLCT) is global network of change-makers,
+				supporters, and allies who support native and indigenous communities
+				to thrive authentically and on their own terms with digital technology
+				both now and in the future.
+			</p>
+			<!-- about us "button" visible on mobile style only -->
+			<div>
+				<button class="button3">About US</button>
+			</div>
+			<!-- "btn-container" holds the other two buttons for the hero section -->
+			<!-- these buttons are "hidden" for mobile and only visible for tablet and larger -->
+			<div class="hidden btn-container">
+				<button class="hidden button1">Our Impact</button>
+				<button class="hidden button1">Our Values</button>
+			</div>
+		</div>
+	</section>
+
+	<!-- Main content -->
+
+	<!-- =======================
+		Community Partnerships Section
+		========================== -->
+
+	<!-- "community-partnerships" is the tag used to target the entire section-->
+	<!-- global class of "wrapper" is applied -->
+	<section class="community-partnerships wrapper">
+		<!-- "community" contains the image and the "partnerships-paragraph" -->
+		<div class="community align-text">
+			<!-- image source set used here - community partnerships.jpg and partnerships-mobile.jpg-->
+			<picture class="card-img">
+				<source
+					media="(min-width:1025px)"
+					srcset="./assets/images/pages/home/community-partnerships.jpg" />
+				<img
+					src="./assets/images/pages/home/mobile/partnerships-mobile.jpg"
+					alt="a group of people" />
+			</picture>
+			<!-- "partnerships-paragraph" contains the header, paragraph text, and button on amaranth background-->
+			<div class="partnerships-paragraph">
+				<h4>Community Partnerships</h4>
+
+				<p>
+					We partner with communities and create aligned programming and
+					events that amplify the voices, experiences, and excellence of
+					native and indigenous populations throughout the world.
+				</p>
+
+				<button class="button1">Learn More</button>
```

Comment:
This button is breaking on tablet. I think it has too much padding, I have been seeing this in a few places. Feel free to edit the global style if you need to.

<img width="571" alt="Screen Shot 2024-01-22 at 11 48 15 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/41a86c83-f58c-4d80-b442-a23069a5c2e7">

Reply:
removed padding from button 1 in global.css and it resolved the issue